packages/rum-core/src/domain/action/listenActionEvents.spec.ts

```diff
@@ -246,7 +246,7 @@ describe('listenActionEvents', () => {
     beforeMouseUp,
     target = document.body,
     clickEventIsPrimary = undefined,
-  }: { beforeMouseUp?(): void; target?: Node; clickEventIsPrimary?: boolean } = {}) {
+  }: { beforeMouseUp?(this: void): void; target?: Node; clickEventIsPrimary?: boolean } = {}) {
```

Comment:
**👏 praise:** ‏TIL! We usually solve this by changing the type to `beforeMouseUp?: (...) => void` but `this:void` sounds nice too